package.json

```diff
@@ -80,14 +80,17 @@
   "license": "AGPL-3.0",
   "scripts": {
     "build": "fedx-scripts webpack",
-    "build:with-theme": "paragon install-theme && npm run build",
+    "build:with-theme": "THEME=npm:@edx/brand-edx.org@latest npm run install-theme && npm run build",
```

Comment:
[inform] When Paragon is re-upgraded to v21, we can use the `paragon` CLI again.